webservice/src/main/resources/app/scripts/containers/Registry-Services/SchemaRegistry/SchemaInfoForm.jsx

```diff
@@ -128,7 +128,9 @@ export default class SchemaFormContainer extends Component {
       var file = e.dataTransfer.files[0];
       var reader = new FileReader();
       reader.onload = function(e) {
-        if(Utils.isValidJson(reader.result)) {
+        if(this.state.type == 'avro' && Utils.isValidJson(reader.result)) {
```

Comment:
can you convert the type or lowerCase and then check with 'avro', just want to make sure we don't run into an issue if by any chance the type comes as upperCase in future.
This change needs to be addressed at various places.

Reply:
I have addressed the review changes.